Libraries/LibC/stdlib.cpp

```diff
@@ -382,6 +382,17 @@ char* mktemp(char* pattern)
     return pattern;
 }
 
+int mkstemp(char* pattern)
+{
+    char* path = mktemp(pattern);
+
+    int fd = open(path, O_RDWR | O_CREAT | O_EXCL, S_IRUSR | S_IWUSR); // I'm using the flags I saw glibc using.
+    if (fd >= 0)
+        return fd;
+
+    return -1;
```

Comment:
Shouldn't this function retry at least a couple of times as long as `open()` fails with `EEXIST`?

Reply:
Let's do this separately from the PR. This is good enough for now, since it will still fail if someone else creates the file in the race window.